omnigibson/__init__.py

```diff
@@ -182,25 +182,26 @@ def start():
     app, sim, Environment, REGISTERED_SCENES, REGISTERED_OBJECTS, REGISTERED_ROBOTS, REGISTERED_CONTROLLERS, \
         REGISTERED_TASKS, ALL_SENSOR_MODALITIES = start()
 
-# Create and expose a temporary directory for any use cases. It will get destroyed upon omni
-# shutdown by the shutdown function.
-tempdir = tempfile.mkdtemp()
+    # Create and expose a temporary directory for any use cases. It will get destroyed upon omni
+    # shutdown by the shutdown function.
+    tempdir = tempfile.mkdtemp()
 
 def shutdown():
-    global app
-    global sim
-    sim.clear()
-    # TODO: Currently tempfile removal will fail due to CopyPrim command (for example, GranularSystem in dicing_apple example.)
-    try:
-        shutil.rmtree(tempdir)
-    except PermissionError:
-        log.info("Permission error when removing temp files. Ignoring")
-    from omnigibson.utils.ui_utils import suppress_omni_log
-    log.info(f"{'-' * 10} Shutting Down {logo_small()} {'-' * 10}")
-
-    # Suppress carb warning here that we have no control over -- it's expected
-    with suppress_omni_log(channels=["carb"]):
-        app.close()
+    if not (os.getenv("OMNIGIBSON_NO_OMNIVERSE", 'False').lower() in {'true', '1', 't'}):
```

Comment:
Nit: put this in a single place instead of checking it in two places. `NO_OMNIVERSE = os.getenv...` and use the NO_OMNIVERSE variable later (in fact maybe put that in gm)

Reply:
done

Reply:
@ChengshuLi @cremebrule Do you think I should put it in gm?

Reply:
No, i think this is fine. It's not really a macro you'd set imho!